### PR TITLE
completed frontend structure 20/20

### PR DESCRIPTION
**Pull request by paulazenanguyen**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+
+class Footer extends Component{
+  render(){
+    return(
+      <>
+        <footer>
+          <div class="footer-content">
+          <h3>Foolish Developer</h3>
```
Comment (WalkerTexas-coder) [inline src/components/Footer.js]: please tab this h3 over

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              </a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                <a class="dropdown-item" href="#">Action</a>
+                <a class="dropdown-item" href="#">Another action</a>
+                <a class="dropdown-item" href="#">Something else here</a>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
```
Comment (WalkerTexas-coder) [inline src/components/Header.js]: Tab this over once please // Check for correct indentation

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import {Nav, NavItem, NavLink} from 'reactstrap';
+
+class Header extends Component{
+  render(){
+    return(
+      <>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (WalkerTexas-coder) [inline src/components/Header.js]: Pretty sure this needs to be capital "N" Nav

**Review by WalkerTexas-coder — COMMENTED** [review]
Hey clean up some indentation and spelling issues and you'll be set for approval! Ping me on slack when it's done.

File: src/components/Footer.js (hunk 1 above)
Comment (JLODI) [reply src/components/Footer.js]: added the indent for h3

File: src/components/Header.js (hunk 2 above)
Comment (JLODI) [reply src/components/Header.js]: lines 8 to 33 are tabbed over one

File: src/components/Header.js (hunk 3 above)
Comment (JLODI) [reply src/components/Header.js]: Capitalizing the nav tags to Nav caused the bootstrap code to not render properly and made styling and links disappear. Kept the tags lower cased.
Screen Shot 2022-04-05 at 3 37 25 PM

**Review by WalkerTexas-coder — APPROVED** [review]
Looks a lot better!

**Pull request merged**